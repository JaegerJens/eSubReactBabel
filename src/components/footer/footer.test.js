import React from 'react';
import Footer from './footer.jsx';
import renderer from 'react-test-renderer';

describe('Footer component', () => {
    it('matches snapshot', () => {
        expect(Footer()).toMatchSnapshot();
    });
});


