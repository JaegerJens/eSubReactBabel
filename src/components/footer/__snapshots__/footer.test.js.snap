exports[`Footer component matches snapshot 1`] = `
<p>
  Version 0.0.1
</p>
`;
